nextjs-app/package.json

```diff
@@ -21,7 +21,16 @@
     "next": "13.4.19",
     "react": "18.2.0",
     "react-dom": "18.2.0",
-    "typescript": "5.2.2",
-    "sass": "1.66.1"
+    "sass": "1.66.1",
+    "typescript": "5.2.2"
+  },
+  "devDependencies": {
+    "deepmerge": "4.3.1"
+  },
+  "resolutions": {
```

Comment:
This has to be done to get around a bug introduced by `@lit-labs/ssr-client`.

I'll demo this to the team / guild hopefully later in the week!